compilers/concrete-compiler/compiler/lib/Analysis/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+add_compile_options(-fsized-deallocation)
```

Comment:
Is there any code actually using sized deallocation?

Reply:
I think this is here to correct some bug that occured because of capnp. Maybe @mkmks can tell more about that ?

Reply:
Without setting this flag, a `delete` implementation can't be found, and the build fails. I don't know enough about C++ semantics to be able to explain what's going on here further.